solr/solr-ref-guide/src/indexing-nested-documents.adoc

```diff
@@ -232,6 +232,8 @@ Indexing nested documents _requires_ an indexed field named `\_root_`:
 <field name="_root_" type="string" indexed="true" stored="false" docValues="false" />
 ----
 
+*If you add this field to to an index that already has data, you will corrupt Solr!*
```

Comment:
"you will corrupt Solr" feels both imprecise and maybe a bit hyperbolic, so maybe it should just say adding this field would require a reindex, and then link to reindexing.adoc#changes-that-require-reindex?